com.unity.render-pipelines.high-definition/Runtime/Material/StackLit/StackLitRayTracing.hlsl

```diff
@@ -74,12 +74,24 @@ void FitToStandardLit( BSDFData bsdfData
                         , uint2 positionSS
                         , out StandardBSDFData outStandardlit)
 {
-    outStandardlit.baseColor = bsdfData.diffuseColor;
-    outStandardlit.specularOcclusion = bsdfData.specularOcclusionCustomInput;
+    // TODO: There's space for doing better here:
+
+    // bool hasCoatNormal = HasFlag(surfaceData.materialFeatures, MATERIALFEATUREFLAGS_STACK_LIT_COAT)
+    //                     && HasFlag(surfaceData.materialFeatures, MATERIALFEATUREFLAGS_STACK_LIT_COAT_NORMAL_MAP);
+    // outStandardlit.normalWS = hasCoatNormal ? surfaceData.coatNormalWS : surfaceData.normalWS;
+    // Using coatnormal not necessarily better here depends on what each are vs geometric normal and the coat strength
+    // vs base strength. Could do something with that and specular albedos...
     outStandardlit.normalWS = bsdfData.normalWS;
-    outStandardlit.perceptualRoughness = bsdfData.perceptualRoughnessA;
+
+    // StandardLit expects diffuse color in baseColor:
+    outStandardlit.baseColor = bsdfData.diffuseColor;
     outStandardlit.fresnel0 = bsdfData.fresnel0;
-    outStandardlit.coatMask = bsdfData.coatMask;
+    outStandardlit.specularOcclusion = 1; // TODO
```

Comment:
Why remove the custom specular occlusion input?

Reply:
In all SO modes where it is non custom, the custom input is initialized to 0. Since the other modes are not handled/distilled for now, the simple fix is to just avoid SO in the reflections for now (otherwise, an SO of 0 will obviously block all specular indirect)